src/main/scala/com/gu/certw/services/CertificateService.scala

```diff
@@ -0,0 +1,73 @@
+package com.gu.certw.services
+
+import java.net.URI
+import java.time.ZonedDateTime
+
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService
+import com.amazonaws.services.simpleemail.model._
+import com.gu.certw.Logging
+import com.gu.certw.models._
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{ JsPath, Json, Reads }
+
+class CertificateService(httpClient: HttpClient, ses: AmazonSimpleEmailService) extends Logging {
+
+  private implicit val prismAcmCertificateParser: Reads[Certificate] = (
+    (JsPath \ "arn").read[String] and
+    (JsPath \ "domainName").read[String] and
+    (JsPath \ "status").read[String] and
+    (JsPath \ "inUseBy").read[List[String]] and
+    (JsPath \ "notAfter").readNullable[ZonedDateTime] and
+    (JsPath \ "meta" \ "origin" \ "ownerId").read[String] and
+    (JsPath \ "meta" \ "origin" \ "accountName").read[String])(Certificate.apply _)
+
+  def listCertificates: List[Certificate] = {
+    val body = httpClient.get(new URI("https://prism.gutools.co.uk/acm-certificates"))
+    (Json.parse(body) \ "data" \ "acmCertificates").as[List[Certificate]]
+  }
+
+  private val activeOrExpired = Set("ISSUED", "EXPIRED")
+
+  def soonToExpire(certificates: List[Certificate], now: ZonedDateTime): List[Certificate] = certificates
+    .filter(cert => activeOrExpired.contains(cert.status))
+    .filter(_.notAfter.forall(_.isBefore(now.plusDays(29))))
```

Comment:
I see 29 hard-coded in a few places. Would it be better to expose this from one location?